github/event_types.go

```diff
@@ -228,6 +228,16 @@ type Discussion struct {
 	Body               *string             `json:"body,omitempty"`
 }
 
+// DiscussionCommentEvent represents discussion comment event
+type DiscussionCommentEvent struct {
+	Action     *string            `json:"action,omitempty"`
+	Comment    *DiscussionComment `json:"comment,omitempty"`
+	Discussion *Discussion        `json:"discussion,omitempty"`
+	Repo       *Repository        `json:"repository,omitempty"`
+	Org        *Organization      `json:"organization,omitempty"`
+	Sender     *User              `json:"user,omitempty"`
```

Comment:
```suggestion
	User     *User              `json:"user,omitempty"`
```

Reply:
<img width="807" alt="Screenshot 2022-10-19 at 1 08 45 AM" src="https://user-images.githubusercontent.com/54154054/196527800-ab2a1e23-fb23-4bed-a8aa-f9f654480170.png">

The Field is mentioned as `Sender` in the Github API docs. Should I change `Sender` -> `User` ??

Reply:
Yes, the JSON makes must match.